internal/services/controller/controller.go

```diff
@@ -471,6 +477,24 @@ func (c *Controller) informerIsAllowedToAccessResource(ctx context.Context, name
 func (c *Controller) Start(done <-chan struct{}) {
 	c.informerFactory.Start(done)
 }
+func processApiResourcesError(log logrus.FieldLogger, err error) map[schema.GroupVersion]bool {
+	cleanedString := strings.Split(err.Error(), "unable to retrieve the complete list of server APIs: ")[1]
+
+	// Split the string by comma
```

Comment:
nitpick: not add any value to context

It would be okay if you used more extensive comments in code but, having only comment with an explanation of what `strings.Split` does not that helpful